src/main/java/eu/donals/Main.java

```diff
@@ -0,0 +1,46 @@
+package eu.donals;
+
+import org.json.simple.JSONArray;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) throws IOException, ParseException, InterruptedException {
```

Comment:
I could've missed them, but I don't think you catch these exceptions right now when they are thrown. For assignment 3 consider catching them so your program doesn't crash when these exceptions occur.